### PR TITLE
Bump up simorgh's packages && version

### PR DESCRIPTION
**Pull request by RayNjeri**
Signed-off-by: Rachael Njeri <rachael.njeri@andela.com>

Resolves #2861 #2860 #2859 #2857 #2856 #2855 #2854 #2852 #2539

**Overall change:** _Bump's up simorgh's packages && version_

**Code changes:**

 _Updates the following Simorgh's Packages:._
- terser-webpack-plugin
- @bbc/psammead-timestamp-container
- @storybook/addon-knobs
- @storybook/theming
- snyk
- speculate
- @bbc/psammead-sitewide-links
- @bbc/psammead-figure
- testing-library/react from

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: package.json
Comment (dr3) [inline package.json]:
```suggestion
  "version": "1.0.0",
```
I dont think were bumping simorgh yet

Comment (jamesdonoh) [reply package.json]: Yes this should stay as-is.

**Review by sareh — APPROVED** [review]
👍 Looks good

**Review by twinlensreflex — APPROVED** [review]
looks good to me 🤖

**Comment by jamesbrumpton** [comment]
This is currently blocked until https://github.com/bbc/simorgh/pull/2800 has been merged. Until then we are unable to sign off Storybook due to the console warnings.

**Comment by jamesbhobbs** [comment]
Good to merge

**Pull request merged**
